pkg/cmd/pulumi/preview.go

```diff
@@ -223,6 +250,13 @@ func newPreviewCmd() *cobra.Command {
 	cmd.PersistentFlags().BoolVar(
 		&configPath, "config-path", false,
 		"Config keys contain a path to a property in a map or list to set")
+	if hasExperimentalCommands() {
+		cmd.PersistentFlags().StringVar(
+			&planFilePath, "save-plan", "",
+			"Save the operations proposed by the preview to a plan file at the given path")
+	}
```

Comment:
Nit: rather than conditionally defining this command based on `hasExperimentalCommands`, I have a slight preference for hiding it based on `hasExperimentalCommands`: that approach doesn't require setting the envvar to use the command, only to discover it.